JustSaying/Messaging/MessageHandling/IMessageContextAccessor.cs

```diff
@@ -0,0 +1,7 @@
+namespace JustSaying.Messaging.MessageHandling
+{
+    public interface IMessageContextAccessor
```

Comment:
If there are going to be 2 interfaces, then shall we mark this one be internal?

Reply:
Needs to be public for DI and utility in others' tests.